api/proto/disperser/v2/disperser_v2.proto

```diff
@@ -27,13 +27,30 @@ service Disperser {
 
   // GetPaymentState is a utility method to get the payment state of a given account, at a given disperser.
   // EigenDA's payment system for v2 is currently centralized, meaning that each disperser does its own accounting.
+  // As reservation moves to be quorum specific and served by permissionless dispersers, GetPaymentState will soon be deprecate
```

Comment:
Do we have a procedure for deprecating grpc methods?

Reply:
no written procedure afaik, so the current plan is for disperser to serve both old and new endpoints. I think we should deprecate this before releasing permissionless dispersers. I can bring this up in the standup for making a standard procedure